data_handler/handlers/health_ratio_level/nostra_alpha.py

```diff
@@ -0,0 +1,25 @@
+from db.models import HealthRatioLevel
+
+from handler_tools.constants import ProtocolIDs
+from handlers.liquidable_debt.values import TIMESTAMP_FIELD_NAME, USER_FIELD_NAME, HEALTH_FACTOR_FIELD_NAME
+
+from health_ratio_handlers import NostrAlphaHealthRatioHandler
+
+
+def run():
+    handler = NostrAlphaHealthRatioHandler()
+
+    data = handler.calculate_health_ratio()
+
+    for id_, health_ratio in data.items():
```

Comment:
as far as you don't user `id_` I think it makes even more sense to use list of dicts